lib/tasks/schools.rake

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :schools do
+  desc "Send nomination invitations to schools"
+  task send_invites: :environment do
+    InviteSchool.run(ARGV[1..-1])
```

Comment:
I think this will take URNs, not IDs?

Reply:
Agree! Probably urns are more meaningful than our ids

Reply:
Are we planning to trigger nomination emails through a rake task, that gets passed a list of URNs?

I don't particularly like the idea of devs scheduling a task and passing a thousand urns to it. I kind of thought that we would have a task that fires off all of them, and afterwards the admins could resend the emails for schools that didn't get it.